app/views/vzory/cookie-banner.njk

```diff
@@ -0,0 +1,697 @@
+    <div class="idsk-cookie-banner " data-nosnippet role="region" aria-label="Cookies na [názov stránky/služby]">
+        <div class="idsk-cookie-banner__message idsk-cookie-banner__example govuk-width-container">
+    
+        <div class="govuk-grid-row">
+            <div class="govuk-grid-column-two-thirds">
+            <h2 class="idsk-cookie-banner__heading govuk-heading-m">Cookies na [názov stránky/služby]</h2>
+    
+            <div class="idsk-cookie-banner__content">
+                <p>Používame základné cookies súbory aby mohla táto stránka fungovať.</p>
+                <p>Taktiež používame dodatočné súbory cookies, akceptovaním súhlasíte s ich používaním.</p>
+            </div>
+            </div>
+        </div>
+    
+        <div class="idsk-button-group">
+            <button value="accept" type="button" name="cookies" class="idsk-button" data-module="idsk-button">
+            Prijať dodatočné cookies
+            </button>
+            <button value="reject" type="button" name="cookies" class="idsk-button" data-module="idsk-button">
+            Odmietnuť dodatočné cookies
+            </button>
+            <a class="govuk-link" href="#" title="Nastavenia cookies">Nastavenia cookies</a>
+        </div>
+        </div>
+    
+        <div class="idsk-cookie-banner__message idsk-cookie-banner__example govuk-width-container" hidden>
+    
+        <div class="govuk-grid-row">
+            <div class="govuk-grid-column-two-thirds">
+    
+            <div class="idsk-cookie-banner__content">
+                <p>Akceptoval si dodatočné súbory cookies. Kedykoľvek ich môžeš zmeniť v <a class="govuk-link" href="#" title="nastaveniach">nastaveniach</a>.</p>
+            </div>
+            </div>
+        </div>
+    
+        <div class="idsk-button-group">
+            <a href="#" role="button" draggable="false" class="idsk-button" data-module="idsk-button">
+            Skryť túto správu
+            </a>
+        </div>
+        </div>
+    
+        <div class="idsk-cookie-banner__message idsk-cookie-banner__example govuk-width-container" hidden>
+    
+        <div class="govuk-grid-row">
+            <div class="govuk-grid-column-two-thirds">
+    
+            <div class="idsk-cookie-banner__content">
+                <p>Odmietol si dodatočné súbory cookies. Kedykoľvek ich môžeš zmeniť v <a class="govuk-link" href="#" title="nastaveniach">nastaveniach</a>.</p>
+            </div>
+            </div>
+        </div>
+    
+        <div class="idsk-button-group">
+            <a href="#" role="button" draggable="false" class="idsk-button" data-module="idsk-button">
+            Skryť túto správu
+            </a>
+        </div>
+        </div>
+    </div>
+    
+</form>
+                {% endset %}
+                {{ exampleComponentHtml | safe }}
+            </div>
+
+            <pre class=" language-markup"><code class="govuk-!-font-size-16  language-markup">{{ exampleComponentHtml }}</code></pre>
```

Comment:
na viac mestiach mas tiez zbytocne prazdne miesta v classe na zaciatku `class=" abc"`, alebo mas tam 2 medzery medzi classami (to su fakt len malickosti, neberem to ako error/warning len ako info)